DLLPasswordFilterImplant/passwordFilter.c

```diff
@@ -41,7 +43,8 @@ BOOL APIENTRY DllMain(HANDLE hModule, DWORD  ul_reason_for_call, LPVOID lpReserv
 
 __declspec(dllexport) BOOLEAN WINAPI InitializeChangeNotify(void)
 {
-	if (IS_DEBUG) {
+	if (IS_DEBUG)
+	{
```

Comment:
Opinion voiced :wink:

Reply:
Just following the surrounding code ;P I resisted the temptation to `%s\t/    /g`